command2.c

```diff
@@ -974,12 +974,7 @@ int compress, force;
     ok = (ynq1() == 'y');
   }
   if (force || ok) {
-    print1("Enter savefile name: ");
-    strcpy(fname,msgscanstring());
-    if (fname[0] == '\0') {
-      print1("No save file entered - save aborted.");
-      ok = FALSE;
-    }
+    gen_save_fname(fname);
```

Comment:
from change to function in `aux1.c` would change to `gen_save_fname( fname, 100 );`

Possible problem with "magic number" of `100` length here but `84` in `aux1.c @p_death`

Ignorable if we don't use `snprintf` in `gen_save_fname`